### DISCUSSION TITLE
goaccess not open results in web page ( .html)

### DISCUSSION THREAD

**Discussion by kamil9966** [allinurl/goaccess#2518] [Q&A]
After using  -  goaccess '/home/kali/Desktop/jira/jira/nginx/backup/jira.mygroup.ru.access.log_05312023_1736' -a -o report.html --log-format=COMBINED not opens web page with results of parsing.
What im doing wrong?
CLI writes Cleaning up resources... after dont have any results
![Снимок экрана 2023-06-02 153250](https://github.com/allinurl/goaccess/assets/52826784/4e8e340b-89b6-4b6a-a4f2-a39e7f1ff327)

**Comment by kamil9966** [comment]
zcat -f /home/kali/Desktop/jira/jira/nginx/backup/jira.mygroup.ru.access.log_05312023_1736* | goaccess -a -o report.html     - not help too

**Comment by allinurl** [comment]
To open the report.html file in your browser, you can use Ctrl+O and provide the file path `/home/kali/Desktop/jira/jira/nginx/backup/report.html`.

**Reply by kamil9966** [reply]
cli tells Fatal error has occurred
Error occurred at: src/output.c - output_html - 1228
Unable to open HTML file: Permission denied.

**Reply by allinurl** [reply]
The error message you received means that you do not have permission to write to the folder `/home/kali/Desktop/jira/jira/nginx/backup/`. To fix this, please make sure that the user running GoAccess has write permissions to this folder.
